wp-content/themes/core/assets/js/theme/ready.js

```diff
@@ -33,6 +34,10 @@ const init = () => {
 
 	bindEvents();
 
+	// initialize on scroll animations
+
+	animateOnScroll();
```

Comment:
:bulb: might be nice to have a global theme setting so this can be optionally turned off for sites that don't want it.